### PR TITLE
[READY] Attempts to modularize shitcode + Removes Fulp mentions

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

This PR was originally gonna be an idea I had to be an alt to #6, but then I saw how shitty our code is. This modularizes it, while removing almost all mentions of Fulp, because this is Tegu, not Fulp. It also fixes a few problems that had with the shitcode conflicting with TG files, and some spelling errors.
After this, I'd like to target mediborgs/secborgs, and SalChems. They're absolute piles of shitcode that deserves eradication from this timeline.

## Why It's Good For The Game

uh... bugs... and shitcode... and stuff, I guess.

## Changelog
:cl: JohnWillard
add: A README.md for all people looking to contribute to Tegustation
add: CMO and CE's medalboxes, we originally had them, but apparently just dissapeared overtime of not being maintained.
fix: Borgs wont have double the comms channels and drop a borg encryption key on deconstruction anymore
fix: Synthflesh factories now work again
remove: Charcoal, SalSynthflesh, and a few other bits of bad and/or unused code
fix: Beefmen customization options now work again
balance: Beefmen are updated to latest version, so they no longer bleed out and no longer fall asleep
fix: Deputies can now get civilian bounties
tweak: Deputies now get sunglasses instead of regular sec huds, who thought not giving sec personnel flash protection was a good idea? Weren't detectives enough for you sickos?
fix: Deputies now get proper departmental access and no longer spawn on the arrivals shuttle
balance: HoP can no longer be most antags
tweak: CMO's improvement kit as given in #1 was moved to the CMO's locker
code: Improved the state of the Tegu code greatly, making it easier to code with less merge conflicts.
/:cl:

### CODE REVIEW

File: ModularTegustation/tegu_overwrite_vars.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 /*
-		WELCOME TO THE FULPSTATION CODE Z-LEVEL!
+		WELCOME TO THE TEGUSTATION CODE Z-LEVEL!
```
Comment (EgorDinamit) [inline ModularTegustation/tegu_overwrite_vars.dm:3]: As said in the file - it is supposed to be at the very bottom of the code/ folder. Put it in its own file starting with "z", so it loads the last, effectively overwriting previous .dm files.

**Comment by EgorDinamit** [comment]
It's supposed to be in its own folder, not "ModularTegustation"...
It's okay when it's in the very bottom of .dme

**Comment by EgorDinamit** [comment]
Time to break all the other 5 PRs.

File: code/modules/vending/clothesmate.dm [hunk 2]
```diff
@@ -109,8 +109,8 @@
 		            /obj/item/clothing/suit/apron/purple_bartender = 2,
 		            /obj/item/clothing/under/rank/civilian/bartender/purple = 2,
 		            /obj/item/clothing/suit/toggle/suspenders/blue = 2,
-		            /obj/item/clothing/shoes/sandal/digitigrade = 1, //fulp
-					/obj/item/clothing/shoes/sneakers/brown/digitigrade = 3, //fulp
+		            /obj/item/clothing/shoes/sandal/digitigrade = 1, // TEGU
+					/obj/item/clothing/shoes/sneakers/brown/digitigrade = 3, // TEGU
```
Comment (EgorDinamit) [inline code/modules/vending/clothesmate.dm:113]: Inconsistent indentation..?

**Comment by JohnFulpWillard** [comment]
> It's supposed to be in its own folder, not "ModularTegustation"...
> It's okay when it's in the very bottom of .dme
> 
> Actually I dunno, it is in the bottom now.

The ModularTegustation folder itself is specifically set to be the last in the tgstation.dme, so it should be fine

**Comment by EgorDinamit** [comment]
Let's fucking go.

**Pull request merged**